src/vcpkg-configuration.json

```diff
@@ -0,0 +1,25 @@
+{
+  "overlay-triplets": [
+    "./triplets"
+  ],
+  "default-registry": {
+    "kind": "git",
+    "baseline": "b322364f06308bdd24823f9d8f03fe0cc86fd46f",
+    "repository": "https://github.com/microsoft/vcpkg"
+  },
+  "registries": [
+    {
+      "kind": "artifact",
+      "location": "https://github.com/microsoft/vcpkg-ce-catalog/archive/refs/heads/main.zip",
+      "name": "microsoft"
+    },
+    {
+      "kind": "git",
+      "repository": "https://github.com/AntaresSimulatorTeam/antares-vcpkg-registry",
+      "baseline": "3fa279503f88d764778bcaacdacd3aa27e323025",
+      "packages": [
+        "sirius-solver"
```

Comment:
Please use antares-integration-v1.7 (currently v1.5) on that registry

Reply:
I've updated the registry with sirius 1.7